datalad/support/tests/test_sshconnector.py

```diff
@@ -74,3 +74,39 @@ def test_ssh_manager_close():
 
     ok_(not exists(opj(manager.socket_dir, 'localhost')))
     ok_(not exists(opj(manager.socket_dir, 'datalad-test')))
+
+
+@skip_ssh
+@with_tempfile(mkdir=True)
+@with_tempfile(content="one")
```

Comment:
Could you please test with that obscure file name have mentioned... git grep obscure to get some examples